metals/src/main/scala/scala/meta/internal/metals/doctor/DoctorResults.scala

```diff
@@ -20,7 +20,7 @@ final case class DoctorResults(
 
 object DoctorResults {
   // Version of the Doctor json that is returned.
-  val version = 4
+  val version = 5
```

Comment:
Should we bump metals to 1.1.x in this case? What do you think @ckipp01 ?

Reply:
According to https://scalameta.org/metals/docs/contributors/releasing#tag-the-release _probably_ yea.

Reply:
We will also try to explain all the recent changes in a plugin authors paragraph then